### PR TITLE
{data}[intel/2015b] pydicom 0.9.9 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3762a8713b9f9f5535ec for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/28459d08a7c80390a275 for a full test report.

File: easybuild/easyconfigs/p/pydicom/pydicom-0.9.9-intel-2015b-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+name = 'pydicom'
+version = '0.9.9'
+
+homepage = 'https://github.com/darcymason/pydicom'
+description = """Read, modify and write DICOM files with python code"""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['https://github.com/darcymason/pydicom/archive/']
+sources = ['v%(version)s.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pydicom/pydicom-0.9.9-intel-2015b-Python-2.7.10.eb]: maybe we should start adding a constant for this?

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/p/pydicom/pydicom-0.9.9-intel-2015b-Python-2.7.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/p/pydicom/pydicom-0.9.9-intel-2015b-Python-2.7.10.eb]: patches welcome ;)

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**